locale/en-US/messages.properties

```diff
@@ -1 +1,8 @@
-config.labelFromProperties=Label defined in a properties file
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+config.rule.item.controls.edit.label=✏
```

Comment:
> Should I use icon image instead of character?

I think that we need not to use icon image instead of emoji.
Because I can recognize this pencil emoji character in my Windows 7 and OS X 10.11.1 environments.

@myokoym, if you install `ttf-ancient-fonts-symbola` in your Debian Linux system, this emoji character is still square character?